### PR TITLE
fix: Update README

### PR DESCRIPTION
**Pull request by burakukula**

### CODE REVIEW

**Review by suevalov — COMMENTED** [review]
Let's make all the links to `f36.contentful.com` and we also need to update README of the tokens package.

**Review by thebesson — APPROVED** [review]

**Comment by burakukula** [comment]
@suevalov yeah, I need to also update main README, now I noticed that is also talking about old packages.

**Comment by burakukula** [comment]
@suevalov I think before we do the switch I am not able to point everything to `https://f36.contentful.com/` - check links will fail for some of them, since they only exists in nextjs version (like contribution)

File: packages/forma-36-tokens/README.md [hunk 1]
```diff
@@ -2,46 +2,46 @@
...
 ```css
-@import '@contentful/forma-36-tokens/dist/css/index.css';
+@import '@contentful/f36-tokens/dist/css/index.css';
 ```
 
 #### SCSS
 
 ```scss
-@import '@contentful/forma-36-tokens/dist/scss/index.scss';
+@import '@contentful/f36-tokens/dist/scss/index.scss';
```
Comment (suevalov) [inline packages/forma-36-tokens/README.md:38]: We don't have SCSS anymore

**Review by suevalov — APPROVED** [review]

**Pull request merged**